hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -143,9 +144,11 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       // TODO add lock heart beating for cases where default lock timeout is too low.
       Table tbl;
       if (base != null) {
+        LOG.info("Committing existing table [{}]", fullName);
```

Comment:
I think these changes are too verbose for INFO, although I do think it would be good to have a log message for the commit.

Could you add an INFO log in `BaseMetastoreTableOperations#commit` instead of these? Something like this:

```java
LOG.info("Successfully committed to table {} in {} ms: {}", table.name(), end - start, newMetadataLocation);
```

That way, logging is not too verbose, consistent across implementations, and adds useful info: the time it took to commit.

Reply:
This was a bit more involved. Please check the resulting method changes:
- BaseMetastoreTableOperations did not know about the tableName - added fullName
- BaseMetastoreTableOperations#commit did not know about the newMetadataLocation - added newMetadataLocation as a return value for doCommit
- Added Stopwatch to the used guava classes